src/Vision/gripV2.py

```diff
@@ -0,0 +1,74 @@
+import cv2
+import numpy
+import math
+from enum import Enum
+
+class GripiPipeline:
```

Comment:
Why is the module named `gripV2` and the class `GripiPipeline`?

Reply:
Done: Renamed gripipipeline to gripV2